src/pages/Question.js

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react'
+import Logo from '../components/Logo.js'
+import Header from '../components/Header.js'
+import  {questions} from '../config';
+import Answer from '../components/Answer.js'
+import '../styles/questions.scss';
+import PreviousButton from '../components/PreviousButton.js';
+
+const Question = () => {
+    const categories = ['food', 'sleep', 'sport', 'misc'];
```

Comment:
Maybe this can be moved to config file?

Reply:
Ik heb deze variable verwijderd omdat dit nu bij een question object staat in de config